docs/team/alanlowzies.md

```diff
@@ -1,6 +1,103 @@
+hosted by the module coordinators of CS2113T.
+
+![tP Code Dashboard](../images/ppp/alanlowzies/tPCodeDashboard.png)
+
+A summary of my code contributions are as follows:
+- Wrote most of the `FileManager` API, which allows WerkIt! to read and write application data
+into text files stored on the user's local filesystem 
+([[WIP] Developer Guide]()). 
+- Implemented the functionality for users to create a new workout 
+([User Guide](../UserGuide.md#create-a-workout-workout-new) | [Developer Guide](../DeveloperGuide.md#create-new-workout)).
+- Wrote most of the `UI` API, which contains UI-related elements, including WerkIt!'s banner logo and response messages.
+- Wrote most of the `WerkIt` API, which contains the code to start up WerkIt!, load its components, load saved file data
+  (if any), and finally receive and process user inputs until the user exits the program.
+- Wrote parts the `Parser` API, which parses user inputs into the appropriate `Command` class types as well
+  as check for invalid characters in the user inputs ([Developer Guide](../DeveloperGuide.md#parsing-user-input-and-getting-the-right-command)).
+- Implemented the HashMap data structure in the `WorkoutList` API to allow `PlanList` to reference
+  workouts created by the user ([Developer Guide](../DeveloperGuide.md#design-considerations-for-creating-a-new-workout)).
+- Wrote the `LogHandler` API, which allows the various components in WerkIt! to log into a file
+  stored in the user's local filesystem ([[WIP] Developer Guide]()).
+- Wrote the `Command` abstract class, which is a template class that sets out the main structure of all types
+  of commands in WerkIt! (e.g. `WorkoutCommand`, `ExerciseCommand`, etc.)
+- Wrote most of the `WorkoutCommand` API, which contains a workout-related command that the user has requested
+  WerkIt! to carry out.
+- Wrote some of the exception APIs for WerkIt!, including `InvalidWorkoutException`, `InvalidExcerciseException`, and
+  `UnknownFileException`.
+- Contributed to the `ExerciseList` API, which is mainly done by my team member Haofeng, by adding
+  additional functionality to the API such as populating the `exercise.txt` file with default exercises. The exercises
+  were thought of by Haofeng :)
+- Wrote JUnit test cases for WerkIt! APIs including (but not limited to) `WorkoutCommand`, `FileManager`, `Parser`, etc.
+- Created skeleton codes and packages for the project.
+
+
+### Enhancements Implemented
+- (Developer and User Guides) Replaced Markdown quote blocks with custom CSS boxes for informational and warning boxes.
+  <br/><br/>**Sample:**<br/>
+<span class="box warning">⚠️This is an example of a warning box. Contains advice or instructions that users
+should take note of to avoid issues in the application.</span>
+<span class="box info">🧾  This is an information box. Contains information that may be useful for
```

Comment:
is this supposed to be an image?

Reply:
I originally used an image, but I thought it would be better if I actually coded the boxes.

Would a screenshot be better? I would like your take on this :)